01-algorithms/06-algorithms-dynamic-programming.md

```diff
@@ -1,4 +1,12 @@
-# Dynamic Programming
+# Optional: Dynamic Programming
+
+### !callout-info
+
+## Optional Lesson
+
+This lesson is optional due to its incomplete status. Please feel free to seek out additional resources to augment the material in this lesson.
+
+### !end-callout
```

Comment:
Should we add a note like this as to why Sliding Window is optional?

Reply:
Added, thank you!